crsqlite_extension/README.md

```diff
@@ -0,0 +1,11 @@
+Work in progres CRSlite using SQLite extensions.
+
+Functionality so far:
+  * Creating new table (' CREATE VIRTUAL TABLE tabel_name USING crsqlite(colum1, column2); ')
+ 
+ TODO:
```

Comment:
Another todo item:
- prevent creation of unique indices that are not the primary key